lib/feature_management.rb

```diff
@@ -48,8 +48,9 @@ def self.enable_identity_verification?
     Figaro.env.enable_identity_verification == 'true'
   end
 
-  def self.reveal_usps_code?
-    Rails.env.development? || current_env_allowed_to_see_usps_code?
+  def self.reveal_usps_flow?
+    env = Rails.env
+    env.development? || env.test? || current_env_allowed_to_see_usps_code?
```

Comment:
If you look at the FeatureManagement class, you'll notice that pretty much all the methods depend on a Figaro key. The class mainly provides descriptive method names and allows us to add other conditions besides Figaro values.